pyproject.toml

```diff
@@ -1,3 +1,42 @@
+[project]
+name = "production-stack"
```

Comment:
?, see https://github.com/vllm-project/production-stack/issues/178 , right now this is[vlllm-router](https://github.com/vllm-project/production-stack/blob/main/setup.py#L4)

Reply:
fixed